### PR TITLE
Upgrade go dependencies

### PR DESCRIPTION
**Pull request by harryzcy**

### CODE REVIEW

File: modules/markup/orgmode/orgmode.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func (Renderer) SanitizerRules() []setting.MarkupSanitizerRule {
 // Render renders orgmode rawbytes to HTML
 func Render(ctx *markup.RenderContext, input io.Reader, output io.Writer) error {
 	htmlWriter := org.NewHTMLWriter()
-	htmlWriter.HighlightCodeBlock = func(source, lang string, inline bool) string {
+	htmlWriter.HighlightCodeBlock = func(source, lang string, inline bool, params map[string]string) string {
```
Comment (harryzcy) [inline modules/markup/orgmode/orgmode.go:54]: The function is changed in https://github.com/niklasfasching/go-org/commit/9b56fc914c80f6b8a6f6dde9c19d70ff7e79a89b

**Review by delvh — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Review by silverwind — DISMISSED** [review]

File: models/auth/token.go [hunk 2]
```diff
@@ -154,16 +154,16 @@ func GetAccessTokenBySHA(token string) (*AccessToken, error) {
...
+		has, err := db.GetEngine(db.DefaultContext).ID(id).Get(accessToken)
 		if err != nil {
 			return nil, err
 		}
 		if has {
-			return token, nil
+			return accessToken, nil
 		}
 		successfulAccessTokenCache.Remove(token)
 	}
```
Comment (harryzcy) [inline models/auth/token.go:169]: `token string` is overrode by `AccessToken` type, which I think is a bug. We need the original `token` when removing it from cache.

File: modules/cache/cache_twoqueue.go [hunk 3]
```diff
@@ -155,7 +155,7 @@ func (c *TwoQueueCache) checkAndInvalidate(key any) {
 	}
 	item, ok := cached.(*MemoryItem)
 	if !ok || item.hasExpired() {
-		c.cache.Remove(item)
+		c.cache.Remove(key)
```
Comment (harryzcy) [inline modules/cache/cache_twoqueue.go:158]: I think `key` should be used when removing from cache.

**Review by harryzcy — COMMENTED** [review]
Please help with the review if the changes are correct. The lru cache generics enforce the type, which reveals two potential bugs.

**Comment by techknowlogick** [comment]
Failure looks to be due to the Action using a (now) outdated version of Go, otherwise this is probably fine to merge once the action sorts itself out. I'll set "wait-merge" label, so that giteabot will keep the branch fresh to until the action is updated.

**Review by wxiaoguang — APPROVED** [review]

**Comment by silverwind** [comment]
Should work again now that https://github.com/actions/go-versions/commit/e0a0673ca194cacf6423441d7c9de2d216d146ac is in.

**Comment by silverwind** [comment]
Branch is passing now, we need to check above open questions.

**Review by lunny — APPROVED** [review]

**Pull request merged**